### PR TITLE
interactive API: graphiql

### PR DESCRIPTION
**Pull request by whimsicallyson**
I think this is ready for some real eyes! As far as I can tell, everything works and looks somewhat decent (the base graphiql design isn't great, but I did what I could). Input very welcome!

### CODE REVIEW

**Comment by znicholasbrown** [comment]
@whimsicallyson this is looking nice! One important piece is missing: we need to add limits at every query level (where applicable) to avoid timeouts (and/or browser memory/rendering limits). e.g.:
```gql
query {
  flow_run {
    id
    task_runs {
      id
    }
  }
}
```
will timeout on large tenants unless we add some limits:
```gql
query {
  flow_run(limit: 100) {
    id
    task_runs(limit:100) {
      id
    }
  }
}
```

**Comment by jlowin** [comment]
@whimsicallyson read-only permissions should be enforced on the backend, so I think it will "just work"

File: src/pages/InteractiveAPI/InteractiveAPIgraphiql.vue [hunk 1]
```diff
@@ -0,0 +1,326 @@
...
+$editor-black: #1d252b; // secondaryGrayDark
+$toolbar-black: #21262b;
+$divider-gray: #4e5965;
+$text-gray: #edf0f3; // secondaryGrayLight
+$accent-pink: #fe5196;
+$accent-orange: #f77062;
+$accent-cyan: #2edaff;
+$code-blue: #0073df;
+$code-pink: #da2072;
+$brackets-gray: #8d98a5;
```
[the remark this answers is not in the document]
Comment (whimsicallyson) [reply src/pages/InteractiveAPI/InteractiveAPIgraphiql.vue:78]: that's the syntax I needed! I tried several things, but none worked :)

**Review by znicholasbrown — APPROVED** [review]
This LGTM @whimsicallyson ! Merging so we can test against dev/staging 😄  🎉

**Pull request merged**